.github/actions/checkDeployBlockers/checkDeployBlockers.js

```diff
@@ -0,0 +1,59 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const {GITHUB_OWNER, EXPENSIFY_CASH_REPO} = require('../../libs/GithubUtils');
+
+const run = function () {
+    const octokit = github.getOctokit(core.getInput('GITHUB_TOKEN', {required: true}));
+    const issueNumber = Number(core.getInput('ISSUE_NUMBER', {required: true}));
+    let hasDeployBlockers = false;
+
+    console.log(`Fetching issue number ${issueNumber}`);
+
+    return octokit.issues.get({
+        owner: GITHUB_OWNER,
+        repo: EXPENSIFY_CASH_REPO,
+        issue_number: issueNumber,
+    })
+        .then(({data}) => {
+            console.log('Checking for unverified PRs or unresolved deploy blockers', data);
+            const pattern = /-\s\[\s]/g;
+            const matches = pattern.exec(data.body);
+            hasDeployBlockers = matches !== null;
+            if (hasDeployBlockers) {
+                console.log('An unverified PR or unresolved deploy blocker was found.');
+                return {};
```

Comment:
I think you should just do `core.setOutput('HAS_DEPLOY_BLOCKERS', true);` here and then `return;` or `process.exit(0)` or something, rather than using a global and continuing down the promise chain.

Reply:
+1